ethers/providers/jsonrpc.nim

```diff
@@ -151,6 +154,18 @@ method getGasPrice*(
     let client = await provider.client
     return await client.eth_gasPrice()
 
+method getMaxPriorityFeePerGas*(
+    provider: JsonRpcProvider
+): Future[UInt256] {.async: (raises: [CancelledError]).} =
+    try:
+      convertError:
+        let client = await provider.client
+        return await client.eth_maxPriorityFeePerGas()
+    except ProviderError:
```

Comment:
`eth_maxPriorityFeePerGas` is generated by a `nim-json-rpc` macro and it will not raise a `ProviderError`. Likely you could catch a `JsonRpcError` here instead.

Reply:
Ah true, good point. Sorry I missed that. 👍